app/src/main/res/layout/fragment_browser_tab.xml

```diff
@@ -48,6 +48,19 @@
         tools:listitem="@layout/item_autocomplete_search_suggestion"
         tools:visibility="visible" />
 
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/quickAccessSuggestionsRecyclerView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="?toolbarBgColor"
+        android:backgroundTint="?toolbarBgColor"
+        android:paddingTop="15dp"
+        android:clipToPadding="false"
+        android:elevation="4dp"
```

Comment:
isn't 4dp the default elevation?

Reply:
Not sure. Removing the elevation produces that favorites are not shown when focusing the URL bar. Also, notice that the previous view (suggestions) also uses the same elevation.